projects/adrv9001/common/adrv9001_qsys.tcl

```diff
@@ -96,6 +148,32 @@ add_connection util_adc_pack.if_packed_fifo_wr_sync axi_adc_dma.if_fifo_wr_sync
 add_connection util_adc_pack.if_packed_fifo_wr_data axi_adc_dma.if_fifo_wr_din
 add_connection axi_adc_dma.if_fifo_wr_overflow util_adc_pack.if_packed_fifo_wr_overflow
 
+# adc-dma2
+
+add_instance axi_adc_dma2 axi_dmac
+set_instance_parameter_value axi_adc_dma2 {ID} {1} 
+set_instance_parameter_value axi_adc_dma2 {DMA_DATA_WIDTH_SRC} {32}
+set_instance_parameter_value axi_adc_dma2 {DMA_DATA_WIDTH_DEST} {32} 
```

Comment:
This can be 64 bits wide as the HPS AXI interface to avoid insertion of width conversion logic.